.github/workflows/runner.yml

```diff
@@ -26,9 +26,9 @@ jobs:
         systemd-detect-virt
     - name: Install dependencies
       run: |
-        sudo cat /etc/apt/sources.list
-        sudo sed -i '/deb-src/d' /etc/apt/sources.list
-        sudo sed -i '/^deb /p;s/ /-src /' /etc/apt/sources.list
+        sudo cat /etc/apt/sources.list.d/ubuntu.sources
```

Comment:
You shouldn't need sudo to read the file, I think.  You could remove that sudo in a separate commit.

Reply:
I took care of this in another commit